### ISSUE TITLE
maDLC: can pickle file from deeplabcut.analyze_videos be reimported into multiple individuals labeling?

### ISSUE THREAD

**Issue by MitchFuchs** [DeepLabCut/DeepLabCut#1524]
Hi guys, 

In maDLC ( version = '2.2.0.2')  can the pickle file created by the function deeplabcut.analyze_videos be easily reimported into the multiple individuals labeling step (either through a command line or GUI)? 

I am trying to reduce labelling time by quickly iterating as follow
small dataset labelling --> data training --> analyze video --> bigger dataset labelling --> data training --> analyze video --> bigger dataset labelling etc

Additionally, where can I find the documentation of the pickle file data structure? I can see it partially through scripts such as predict_multianimal.py but your official documentation would be helpful. 

Thanks a lot for your help!

Cheers, 
Mitch.

**Comment by MMathisLab** [comment]
From https://deeplabcut.github.io/DeepLabCut/docs/maDLC_UserGuide.html#decision-break-point

> ..... we recommend extracting and labeling more frames (even from more videos). Try to label close interactions of animals for best performance. Once you label more, you can create a new training set and train.
> 
> You can either:
> 
> extract more frames from existing or new videos and label as when initially building the training data set, or
> 
> extract outlier frames based on the videos you just analyzed. To do this, you first need to analyze a video and convert to tracklets (Step 1 and 2 in the Analyze Video tab of the GUI), then you load the tracklet file into the refine tracklet GUI, and “Save” without making any modifications (or run deeplabcut.convert_raw_tracks_to_h5(config_path, picklefile)). That will create the files needed in the Extract Outlier Frames tab of the GUI.

Alternative path: (also from link above):

`deeplabcut.analyze_videos(`

`deeplabcut.convert_detections2tracklets(`

`deeplabcut.stitch_tracklets(`


In general, we ask you please post questions on the Image Forum, as more people see it / can chip in - we'd prefer to keep github for issues, hence the template we request.
